src/bio/bio_xstream.c

```diff
@@ -1873,6 +1876,10 @@ bio_led_event_monitor(struct bio_xs_context *ctxt, uint64_t now)
 	struct bio_bdev         *d_bdev;
 	int			 rc;
 
+	if (!bio_vmd_enabled) {
```

Comment:
[style] normally if this is only one line we don't need {,
         if (!bio_vmd_enabled)
                      return;